schema/cargo.kdl

```diff
@@ -0,0 +1,172 @@
+@kdl:schema "https://github.com/kdl-org/kdl/blob/main/schema/kdl-schema.kdl"
+
+metadata {
+    // TODO: update this link when we're ready to release something.
+    link "https://github.com/kdl-org/kdl/blob/main/schema/cargo.kdl" rel=self
+    title "Cargo Schema" lang=en
```

Comment:
For the `lang` stuff, I can see the value of having that as a property so that you can easily have mixed-language documents, but would be be useful to just have a `metadata.lang` node that sets the document default unless otherwise specified?

Also, how does one specify the language of a string in a property? Like when you have:
```kdl
node version title="The version of the package."
```
How do you mark the lang of the `title` string?

Again likely my cluelessness: do many other schemas have something like this?

Reply:
I’m ok removing the lang stuff tbh.

Reply:
I'll have a bit more of a ponder about it, but I agree that might be a good way to simplify things a little bit, since making it consistent would perhaps be a lot of work

Reply:
After more thought, I think removing this might be best for the sake of keeping things simple and consistent!

Reply:
removed